### DISCUSSION TITLE
目前还不支持命令行通过v2ctl api 创建用户呢吧

### DISCUSSION THREAD

**Discussion by snail2sky** [v2fly/v2ray-core#2027] [Q&A]
看了下官方的帮助手册，发现有提到可以通过api 创建用户，但是命令行的好像还没有支持？
![image](https://user-images.githubusercontent.com/49757958/193762476-d3efc33e-7852-4c99-8cc7-a594a5efabc9.png)
这里边也没有相应的方法
![image](https://user-images.githubusercontent.com/49757958/193762547-23af7971-a2b5-4e45-8762-10eb0b87c7f2.png)

**Comment by AkinoKaede** [comment]
Yes, you are right.

**Reply by snail2sky** [reply]
好的 感谢

**Comment by hnyyghk** [comment]
i have do some work on it, 
now works well on add/remove vmess user in without config files mode

https://github.com/hnyyghk/v2ray-core

it is difficult to parse config files

**Reply by snail2sky** [reply]
好的  看一下高手(大佬)的操作
